frontend/src/routes/login.lazy.tsx

```diff
@@ -2,6 +2,9 @@ import { useMutation, useQueryClient } from "@tanstack/react-query";
 import { createLazyFileRoute, useNavigate } from "@tanstack/react-router";
 import { RequestInput, apiClient } from "~/api/client";
 import { Input } from "~/components/ui/input";
+import blackLogo from "/logo/ZaneOps-HORIZONTAL-BLACK.svg";
+import logoSymbol from "/logo/ZaneOps-SYMBOL-BLACK.svg";
+import whiteLogo from "/logo/Zane Ops logo white text.svg";
```

Comment:
Can you use a relative import instead ? Or maybe just pass the url to the image, this might not work in production.